admin_app/src/app/dashboard/components/performance/DetailsDrawer.tsx

```diff
@@ -203,83 +213,45 @@ const DetailsDrawer: React.FC<DetailsDrawerProps> = ({
           </Typography>
         </Box>
 
-        <Grid
-          container
-          columns={13}
-          sx={{
-            mt: 2,
-            display: "flex",
-            flexDirection: "row",
-            justifyContent: "center",
-            borderBottom: 1,
-            borderTop: 1,
-            borderColor: "secondary.main",
-            backgroundColor: "lightgray.main",
-            fontSize: "small",
-            fontWeight: 600,
-            lineHeight: "32px",
-          }}
-        >
-          <Grid item md={3} sx={{ px: 1 }}>
-            Timestamp
-          </Grid>
-          <Grid item md={5} sx={{ px: 1 }}>
-            User Question
-          </Grid>
-          <Grid item md={5} sx={{ px: 1 }}>
-            User Feedback
-          </Grid>
-        </Grid>
-        {data.user_feedback.map((feedback) => (
-          <Grid
-            container
-            columns={13}
-            sx={{
-              display: "flex",
-              flexDirection: "row",
-              justifyItems: "left",
-              borderBottom: 1,
-              borderColor: "secondary.main",
-              fontSize: "x-small",
-              lineHeight: "32px",
-              overflow: "hidden",
-              py: 0.5,
-            }}
-          >
-            <Grid item md={3} sx={{ px: 1, lineHeight: "20px" }}>
-              {Intl.DateTimeFormat("en-ZA", {
-                dateStyle: "short",
-                timeStyle: "short",
-              }).format(new Date(feedback.timestamp))}
-            </Grid>
-            <Grid
-              item
-              md={5}
-              sx={{
-                overflow: "hidden",
-                whiteSpace: "nowrap",
-                textOverflow: "ellipsis",
-                px: 1,
-                lineHeight: "20px",
-              }}
-            >
-              {feedback.question}
-            </Grid>
-            <Grid
-              item
-              md={5}
-              sx={{
-                px: 1,
-                overflow: "hidden",
-                whiteSpace: "nowrap",
-                textOverflow: "ellipsis",
-                lineHeight: "20px",
-              }}
-            >
-              {feedback.feedback}
-            </Grid>
-          </Grid>
-        ))}
+        <TableContainer component={Paper} sx={{ mt: 2 }}>
```

Comment:
Just curious, what are the advantages of `TableContainer` and  `Table` compared to `Grid`?

Reply:
We had a bug where cells of each row would resize weirdly smaller than the "s" screen size, also this is the exact intended usecase of this component so makes sense to use it directly!